cli/popper/commands/cmd_run.py

```diff
@@ -120,124 +112,148 @@
     default=popper.scm.get_git_root_folder()
 )
 @pass_context
-def cli(ctx, action, wfile, skip_clone, skip_pull, skip, reuse,
-        recursive, quiet, debug, dry_run, parallel, log_file,
-        with_dependencies, on_failure, runtime, workspace):
-    """Execute github actions workflows.
+def cli(ctx, **kwargs):
+    """Executes one or more pipelines and reports on their status.
     """
-    popper.scm.get_git_root_folder()
+    if os.environ.get('CI') == 'true':
+        # When CI is set,
+        log.info('Running in CI environment...')
+        occurences = parse_commit_message()
```

Comment:
can we rename this to `popper_run_occurrences` or `popper_run_instances`, as a way of making it more explicit what this refers to?